NumLib/ODESolver/EquationSystem.h

```diff
@@ -30,9 +29,7 @@ enum class IterationResult : char
  *
  * \tparam Vector the type of the solution vector of the equation.
```

Comment:
I didn't see it before, but all of the Doxygen documentation needs a fixup; the template parameter does not exist anymore.